subworkflows/local/arg.nf

```diff
@@ -8,17 +8,23 @@ include { AMRFINDERPLUS_RUN                 }  from '../../modules/nf-core/amrfi
 include { DEEPARG_DOWNLOADDATA              }  from '../../modules/nf-core/deeparg/downloaddata/main'
 include { DEEPARG_PREDICT                   }  from '../../modules/nf-core/deeparg/predict/main'
 include { FARGENE                           }  from '../../modules/nf-core/fargene/main'
-include { HAMRONIZATION_ABRICATE            }  from '../../modules/nf-core/hamronization/abricate/main'
-include { HAMRONIZATION_RGI                 }  from '../../modules/nf-core/hamronization/rgi/main'
-include { HAMRONIZATION_DEEPARG             }  from '../../modules/nf-core/hamronization/deeparg/main'
-include { HAMRONIZATION_AMRFINDERPLUS       }  from '../../modules/nf-core/hamronization/amrfinderplus/main'
-include { HAMRONIZATION_FARGENE             }  from '../../modules/nf-core/hamronization/fargene/main'
-include { HAMRONIZATION_SUMMARIZE           }  from '../../modules/nf-core/hamronization/summarize/main'
 include { RGI_CARDANNOTATION                }  from '../../modules/nf-core/rgi/cardannotation/main'
 include { RGI_MAIN                          }  from '../../modules/nf-core/rgi/main/main'
 include { UNTAR as UNTAR_CARD               }  from '../../modules/nf-core/untar/main'
 include { TABIX_BGZIP as ARG_TABIX_BGZIP    }  from '../../modules/nf-core/tabix/bgzip/main'
 include { MERGE_TAXONOMY_HAMRONIZATION      }  from '../../modules/local/merge_taxonomy_hamronization'
+include { ARGNORM as ARGNORM_AMRFINDERPLUS  }  from '../../modules/nf-core/argnorm/main'
+include { ARGNORM as ARGNORM_DEEPARG        }  from '../../modules/nf-core/argnorm/main'
+include { ARGNORM as ARGNORM_ABRICATE       }  from '../../modules/nf-core/argnorm/main'
+include { HAMRONIZATION_RGI                 }  from '../../modules/nf-core/hamronization/rgi/main'
+include { HAMRONIZATION_FARGENE             }  from '../../modules/nf-core/hamronization/fargene/main'
+include { HAMRONIZATION_SUMMARIZE           }  from '../../modules/nf-core/hamronization/summarize/main'
+include { HAMRONIZATION_ABRICATE as HAMRONIZATION_ABRICATE_TSV  }  from '../../modules/nf-core/hamronization/abricate/main'
+include { HAMRONIZATION_ABRICATE as HAMRONIZATION_ABRICATE_JSON }  from '../../modules/nf-core/hamronization/abricate/main'
+include { HAMRONIZATION_DEEPARG as HAMRONIZATION_DEEPARG_TSV    }  from '../../modules/nf-core/hamronization/deeparg/main'
+include { HAMRONIZATION_DEEPARG as HAMRONIZATION_DEEPARG_JSON   }  from '../../modules/nf-core/hamronization/deeparg/main'
+include { HAMRONIZATION_AMRFINDERPLUS as HAMRONIZATION_AMRFINDERPLUS_TSV  }  from '../../modules/nf-core/hamronization/amrfinderplus/main'
+include { HAMRONIZATION_AMRFINDERPLUS as HAMRONIZATION_AMRFINDERPLUS_JSON }  from '../../modules/nf-core/hamronization/amrfinderplus/main'
```

Comment:
```suggestion
include { HAMRONIZATION_ABRICATE            }  from '../../modules/nf-core/hamronization/abricate/main'
include { HAMRONIZATION_AMRFINDERPLUS       }  from '../../modules/nf-core/hamronization/amrfinderplus/main'
include { HAMRONIZATION_DEEPARG             }  from '../../modules/nf-core/hamronization/deeparg/main'
include { HAMRONIZATION_FARGENE             }  from '../../modules/nf-core/hamronization/fargene/main'
include { HAMRONIZATION_RGI                 }  from '../../modules/nf-core/hamronization/rgi/main'
include { HAMRONIZATION_SUMMARIZE           }  from '../../modules/nf-core/hamronization/summarize/main'
```
We can actually get rid of the `.json` output, it is only used for hamronization/summarize downstream, which can be replaced by `.tsv` format.